server/index.js

```diff
@@ -80,6 +80,13 @@ export default class Server {
       const p = join(__dirname, '..', 'client', ...(params.path || []))
       await serveStatic(req, res, p)
     })
+
+    this.router.get('/_webpack/:number', async (req, res, params) => {
```

Comment:
As far as I read https://github.com/zeit/next.js/issues/728, is this intentional ? Anyway it would be better to use a same logic to serve these files on dev and production IMO.